besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -506,9 +507,8 @@ private InetAddress autoDiscoverDefaultIP() {
   @Option(
       names = {"--fast-sync-min-peers"},
       paramLabel = MANDATORY_INTEGER_FORMAT_HELP,
-      description =
-          "Minimum number of peers required before starting fast sync. (default: ${DEFAULT-VALUE})")
-  private final Integer fastSyncMinPeerCount = FAST_SYNC_MIN_PEER_COUNT;
+      description = "Minimum number of peers required before starting fast sync.")
```

Comment:
is it possible to says which is the default pre and post merge? 
since there is actually a default, it seems not correct to remove the info here

Reply:
I modified to add the default value again